mw/diag/dtc/database/dtc_database.hpp

```diff
@@ -24,14 +24,19 @@ namespace dtc{
 
 class DtcDatabase{
  private:
+ /**
+  * @brief kluczem jest id błędu
+  * 
+  */
   std::unordered_map<uint16_t,DtcDatabaseElement> errors_{};
   core::logger::Logger logger_;
+  uint16_t error_id;
```

Comment:
nie czaje tego

Reply:
przecież to dostajesz od klientów

Reply:
to zbieżność nazw, id w sensie numer nie typ, po prostu planowalem to iterować żeby mieć unikalny klucz po którym można znajdywać błędy